jdaviz/configs/cubeviz/plugins/slice/slice.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <j-tray-plugin
     :config="config"
-    plugin_key="Slice"
+    :plugin_key="plugin_key || 'Slice'"
```

Comment:
see https://github.com/spacetelescope/lcviz/pull/133/files#diff-32e4b267e193d9c8832ff3c8b50929e8fc6f755b802f7dad26bd910cc22375cfR42 for how this will be used in lcviz to override the API hint to access the plugin from `plugins['Slice']` to `plugins['Time Selector']`.

Reply:
Looks surprising at first, but super simple in the end. Neat!